### PR TITLE
Add customization second tab.RD-19717

### PR DESCRIPTION
**Pull request by dhouha123**

### CODE REVIEW

File: app/src/main/java/com/dimelo/sampleapp/RcConfig.java [hunk 1]
```diff
@@ -49,7 +49,7 @@ static Dimelo setupDimelo(Context context) {
             dimelo.setUserIdentifier(userIdVal);
         }
 
-        if (isHmsAvailable(context)) {
+      /*  if (isHmsAvailable(context)) {
```
Comment (waelba) [inline app/src/main/java/com/dimelo/sampleapp/RcConfig.java]:
```suggestion
      /*if (isHmsAvailable(context)) {
```

File: app/src/main/java/com/dimelo/sampleapp/chats/TabTelecom.java [hunk 2]
```diff
@@ -90,6 +94,22 @@ private void customize(){
         customisation.dateTextColor = Color.WHITE;
         customisation.hourTimeTextColor = Color.WHITE;
         customisation.createNewThreadBackgroundColor = Color.RED;
+        int paddingReplies = (int) TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, getResources().getDimension(R.dimen.rc_quick_replies_padding), getResources().getDisplayMetrics());
```
Comment (waelba) [inline app/src/main/java/com/dimelo/sampleapp/chats/TabTelecom.java]:
```suggestion
        int paddingReplies = (int)TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, getResources().getDimension(R.dimen.rc_quick_replies_padding), getResources().getDisplayMetrics());
```

File: app/src/main/java/com/dimelo/sampleapp/chats/TabTelecom.java [hunk 3]
```diff
@@ -90,6 +94,22 @@ private void customize(){
...
+        customisation.quickRepliesItemPadding = new RcFragment.Customization.Padding(paddingReplies, paddingReplies, paddingReplies, paddingReplies);
+        SimpleDateFormat dateFormatter = new SimpleDateFormat("yyyy-MM-dd");
+        customisation.dateFormatter = dateFormatter;
+        customisation.messageFont = Typeface.DEFAULT_BOLD;
+        customisation.agentTemplateBorderColor = Color.RED;
+        customisation.agentTemplateWithImageBodyBackgroundColor = Color.GRAY;
+        int quickRepliesHeight = (int) TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, getResources().getDimension(R.dimen.rc_quick_replies_height), getResources().getDisplayMetrics());
```
Comment (waelba) [inline app/src/main/java/com/dimelo/sampleapp/chats/TabTelecom.java]:
```suggestion
        int quickRepliesHeight = (int)TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, getResources().getDimension(R.dimen.rc_quick_replies_height), getResources().getDisplayMetrics());
```

File: app/src/main/res/values/bool.xml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <bool name="dimelo_toolbar_display_title">true</bool>
```
Comment (waelba) [inline app/src/main/res/values/bool.xml:3]: default value of `dimelo_toolbar_display_title` is `true` so you can remove this file or change the value to `false`

**Comment by DrissTM** [comment]
same as for gitlab please use the correct format for the PR title and write a description with a link to the ticket and what have been done

File: app/src/main/res/values/bool.xml (hunk 4 above)
Comment (waelba) [reply app/src/main/res/values/bool.xml:3]: ping @dhouha123

**Pull request closed without merging**